src/card.tsx

```diff
@@ -393,3 +403,296 @@ export const SnapCard = React.forwardRef<HTMLDivElement, SnapCardProps>(
         );
     }
 );
+
+type CardAction = {
+    label: string;
+    onPress: () => void;
+    Icon: React.FC<IconProps>;
+};
+
+type CardActionsGroupProps = {
+    actions: Array<CardAction>;
+    isInverse?: boolean;
+};
+
+const CardActionsGroup = ({actions, isInverse}: CardActionsGroupProps): JSX.Element => {
+    return (
+        <Inline space={0}>
+            {actions.map(({onPress, label, Icon}, index) => (
+                <IconButton
+                    size={48}
+                    key={index}
+                    onPress={onPress}
+                    aria-label={label}
+                    className={styles.cardActionIconButton}
+                    style={{display: 'flex'}}
+                >
+                    <div className={isInverse ? styles.cardActionInverse : styles.cardAction}>
+                        <Icon color={vars.colors.neutralHigh} size={20} />
+                    </div>
+                </IconButton>
+            ))}
+        </Inline>
+    );
+};
+
+type AspectRatio = '1:1' | '16:9' | '7:10' | '4:3' | 'auto';
+
+const CSS_ASPECT_RATIO = {
+    '1:1': '1',
+    '16:9': '16 / 9',
+    '7:10': '7 / 10',
+    '4:3': '4 / 3',
+    auto: 'auto',
+} as const;
```

Comment:
could these values be taken from image/video files? or this are a differents images?

Reply:
@yceballost, @aweell, there is a list of supported aspect ratios, or at least recomended ones? I've copied these ones from Image/Video component, and I think they were specified in the specs, but I don't know if we want to recommend the same aspectRatios for DisplayMediaCards, a different set, or nothing at all

Reply:
Yes, there's a different set of recommended values:

* 9:10
* 1:1
* 16:9
* Custom value

We should use these, instead of the Image defaults.

![Screenshot 2023-02-24 at 10 28 22](https://user-images.githubusercontent.com/44420072/221143482-da27f4d5-6283-4d4f-85e3-e31f299ec43e.png)